proto/cofidectl_plugin/v1alpha1/plugin.proto

```diff
@@ -5,16 +5,119 @@ syntax = "proto3";
 
 package proto.cofidectl_plugin.v1alpha1;
 
+import "proto/ap_binding/v1alpha1/ap_binding.proto";
+import "proto/attestation_policy/v1alpha1/attestation_policy.proto";
+import "proto/federation/v1alpha1/federation.proto";
 import "proto/trust_zone/v1alpha1/trust_zone.proto";
 
 option go_package = "github.com/cofide/cofide-api-sdk/gen/go/proto/cofidectl_plugin/v1alpha1";
 
 service DataSourcePluginService {
+  rpc Validate(ValidateRequest) returns (ValidateResponse);
+
+  rpc GetTrustZone(GetTrustZoneRequest) returns (GetTrustZoneResponse);
   rpc ListTrustZones(ListTrustZonesRequest) returns (ListTrustZonesResponse);
+  rpc AddTrustZone(AddTrustZoneRequest) returns (AddTrustZoneResponse);
+  rpc UpdateTrustZone(UpdateTrustZoneRequest) returns (UpdateTrustZoneResponse);
+
+  rpc AddAttestationPolicy(AddAttestationPolicyRequest) returns (AddAttestationPolicyResponse);
+  rpc GetAttestationPolicy(GetAttestationPolicyRequest) returns (GetAttestationPolicyResponse);
+  rpc ListAttestationPolicies(ListAttestationPoliciesRequest) returns (ListAttestationPoliciesResponse);
+
+  rpc AddAPBinding(AddAPBindingRequest) returns (AddAPBindingResponse);
+  rpc DestroyAPBinding(DestroyAPBindingRequest) returns (DestroyAPBindingResponse);
```

Comment:
Not a strong opinion here, but any specific reason for `Destroy` over `Remove`? (more out of curiosity)

Reply:
Perhaps remove would be a better antonym to add. I went with Destroy in the [DataSource interface](https://github.com/cofide/cofidectl/blob/7afabc8c49a5568f9c977e6e7088dd6179b3df7e/pkg/plugin/interface.go#L26), is if we change it should change there too.

FWIW I went with `del` in the CLI: `cofidectl attestation-policy-binding del ...`